Content/src/Server/ServerGiraffe.fs

```diff
@@ -1,35 +1,44 @@
-open System
-open System.IO
-open System.Threading.Tasks
-
+#if (remoting)
+open Fable.Remoting.Server
+open Fable.Remoting.Giraffe
+#endif
+#if (deploy == "azure")
+open FSharp.Azure.StorageTypeProvider
+#endif
+open FSharp.Control.Tasks.V2
+open Giraffe
+#if (deploy == "azure")
+open global.Shared
```

Comment:
what is global.Shared ?

Reply:
I seem to recall that I had to do this because there was another `Shared` namespace that was clashing with the Shared namespace in the common types file in the template. `global.Shared` just relates to that. I'll check what caused it.